blackbox/docs/administration/auth_methods.txt

```diff
@@ -39,3 +39,24 @@ setting like this:
     auth:
       host_based:
         http_default_user: dustin
+
+
+.. _auth_cert:
+
+Client Certificate Authentication Method
+========================================
+
+When the ``cert`` authentication method is used, the client has to connect to
+CrateDB using SSL with a valid client certificate. For the authentication to
+succeed the ``common name`` of the certificate has to match the provided
```

Comment:
This matches the behaviour from postgres, but I guess for HTTPS we want to use the userName from the commonName if the userName is otherwise not provided?